serverless.yml

```
@@ -2,12 +2,17 @@ service: airkeeper
 
 provider:
   name: aws
-  # region: ${opt:region}
-  # stage: ${opt:stage}
+  region: ${env:REGION}
+  stage: ${env:STAGE}
```

Is this now being used?

---

I set them in the provider so they could be used in the lambda IAM role:
`'arn:aws:lambda:${self:provider.region}:${aws:accountId}:function:airkeeper-${self:provider.stage}-process-subscriptions'`

> Tested it locally and it seems to be working fine but I must admit that it is a bit annoying to have to set valid AWS creds in order to run Airkeeper locally

I think we could add defaults for dev values. This should work:
```suggestion
  region: ${env:REGION, 'us-east-1'}
  stage: ${env:STAGE, 'dev'}
```